humans/utils.py

```diff
@@ -45,4 +47,4 @@ def key_state(key, gpg):
     else:
         state = "valid"
 
-    return key_fingerprint, state
+    return key_fingerprint, (state, days_to_expire)
```

Comment:
Why not just `return key_fingerprint, state, days_to_expire`? It could be easily unpacked this way: `fingerprint, *state = key_state(key)`

Nevertheless the return on this function is getting too complex and in the future we should wrap it in something like a `Key` object, instead of returning 3 values that are related.